usaspending_api/api_contracts/contracts/awards/idvs/Activity.md

```diff
@@ -108,9 +111,11 @@ This endpoint is used to power the IDV (Indefinite Delivery Vehicle) Activity vi
     Unique internal natural identifier for an award.
 + `awarding_agency` (required, string)
 + `awarding_agency_id` (required, number)
-+ `last_date_to_order` (required, string, nullable)
++ `period_of_performance_ultimate_end_date` (required, string, nullable)
```

Comment:
Noting we should remove `nullable` as part of [DEV-3006](https://federal-spending-transparency.atlassian.net/browse/DEV-3006)

Reply:
My understanding that we were using a new flag for DEV-3006, so without the flag it would could still return nulls

Reply:
☝️True